src/utils/TokenUtils.ts

```diff
@@ -45,7 +45,7 @@ export async function estApprove(
  * @param {String} account
  * @param {String} tokenAddress
  * @param {String} spender
- * @param {String} amount  (always expressed as wei)
+ * @param {String} amount amount of ERC20 tokens, expressed without decimals (not as wei)
```

Comment:
why without decimals?

Reply:
but those are not decimals , without decimals means you can send only integers and that is false.